jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,476 @@
+`/project/{project-slug}/envvar` | This endpoint enables you to create an environment variable for a project by making a POST request and passing the `project-slug` parameter in the request.
+`/project/{project-slug}/envvar/{name}` | This endpoint enables you to delete an existing environment variable from a project by making a DELETE request and passing the `project-slug` and `name` parameters in the request.
+`/project/{project-slug}/envvar/{name}` | This endpoint enables you to retrieve a masked value for an environment variable for a project by making a GET request and passing the `project-slug` and `name` parameters in the request.
+
+### Job Endpoints
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`/project/{project-slug}/job/{job-number}` | This endpoint enables you to retrieve job details by making a GET request and passing the project-slug and job-number parameters in the request.
+`/project/{project-slug}/job/{job-number}/cancel` | This endoint enables users to cancel a job by making a POST request and passing the `project-slug` and `job-number` parameters in the request.
+`/project/{project-slug}/{job-number}/artifacts`  | This endpoint enables users to retrieve artifacts for a job by making a GET request and passing the project-slug and job-number parameters in the request.
+`/project/{project-slug}/{job-number}/tests` | This endpoint enables you to retrieve test metadata for a job by making a GET request and passing the `project-slug` and `job-number` parameters in the request.
+
+### Workflow Endpoints
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`/workflow/{id}` | This endpoint enables users to you to return a workflow by making a GET request and passing the `id` parameter in the request.
+`/workflow/{id}/cancel` | This endoint enables you to cancel a workflow by making a POST request and passing the `id` parameter in the request.
+`/workflow/{id}/job`  | This endpoint enables you to return the jobs associated with a workflow by making a GET request and passing the `id` parameter in the request.
+`/workflow/{id}/rerun` | This endpoint enables you to rerun a workflow by making a POST request and passing the `id` parameter in the request.
+
+## API Syntax
+
+When making an API request, make sure you follow standard REST API syntax and formatting. Adhering to proper REST API syntax ensures the API server can properly process your request and return a valid JSON response. To make a request to the CircleCI API, use the following format:
+
+`https://circleci.com/api/v2`
+
+Where:
+
+- `https://circleci.com` is the resource URL for the API being called.
+- `api` is the service being called.
+- `v2` is the API version.
+
+# Getting Started with the API
```

Comment:
This is a stylistic choice, but I'd pull the "getting started" part before the huge list of available endpoints, which imho should probably go at the very end as it's more a reference than an explanation or narrative.

Reply:
Agreed.....went ahead and moved the list of endpoints to the end of the page in a Reference section.